server/tests/database/test_user_data_handler.py

```
@@ -3,21 +3,22 @@
 
     assert isinstance(user, User)
 
     assert user.id == 5
-    assert user.username == "user1"
+    assert user.firstName == "John"
+    assert user.lastName == "Doe"
     assert user.email == "user1@gmail.com"
     assert user.passwordHash == "12345"
 
 
 def test_create_user_invalid_email(app):
```

Ruff (0.15.5)

[warning] Unused function argument: `app`

(ARG001)